src/threshold-ts/applications/index.ts

```diff
@@ -321,21 +321,21 @@ export class Application implements IApplication {
       remainingAuthorizationDecreaseDelay.toString()
     )
 
-    let isDeauthorizationReqestActive = true
+    let isDeauthorizationRequestActive = true
     if (_remainingAuthorizationDecreaseDelay.eq(MAX_UINT64)) {
       // If a `remainingAuthorizationDecreaseDelay` is equal `MAX_UINT64` the
       // deauthorization reqest is pending and an operator have to call
```

Comment:
Let's also fix a similar typo in this comment

```suggestion
      // deauthorization request is pending and an operator have to call
```